src/sentry/static/sentry/app/utils/withApi.tsx

```diff
@@ -46,7 +46,8 @@ const withApi = <P extends InjectedApiProps>(
     private api: Client;
 
     render() {
-      return <WrappedComponent api={this.api} {...(this.props as P)} />;
+      const {api = this.api} = this.props as P;
+      return <WrappedComponent {...(this.props as P)} api={api} />;
```

Comment:
but here you are passing everything twice again.. it's fine as the `api` prop comes after `this.props`, but I I would rather write like:

 ```
 const {api, ...props} = this.props;
 return <WrappedComponent {...({api: api ?? this.api, ...props} as P)} />;
```

Reply:
changed it over to this style